guides/WRITING-DOCUMENTATION.md

```diff
@@ -15,176 +15,147 @@ For a component named "Foo", you would have the following:
         ├── Foo.jsx             React component
         ├── Foo.test.jsx        React component tests
         └── Foo.example.jsx     React component used to render an example
+        └── Foo.example.html    HTML example
 ```
 
-# CSS components
+## CSS components
 
 [KSS](https://github.com/kss-node/kss-node) is the primary way documentation is written. They establish what pages get created and the content on those pages. We use KSS to parse CSS comment blocks to extract documentation, and it expects your comment block to follow this particular format:
 
-```css
+```scss
 /*
 [Heading]
 
 [Description]
 
-[@flags]
-
-Markup:
-[Inline HTML/EJS or filename]
+[Flags]
 
 [Modifiers]
 
+Markup: [filename]
+
 Style guide: [Reference]
+
+[Documentation and guidance]
 */
 ```
 
-### Heading
+### Heading (required)
 
 The heading is used for the title of a generated page or the heading if this is a subsection of a page.
 
-This is a required field, but if you want to create a subsection without a heading, you can enter `---`.
+**note:** If you want to create a subsection without a heading, add `---` after the opening `/*`.
+
+### Description
+
+The description should describe what pattern does or should be used for in plain language.
 
-### @flags
+### Flags
```

Comment:
I know the `@flags` looks weird, but keeping that formatting consistent reduces confusion that it could be some other kind of flag. QPP for example uses "flags" to turn features on and off.